src/options.jl

```diff
@@ -36,6 +36,7 @@ Base.@kwdef struct Options
     yas_style_nesting::Bool = false
     short_circuit_to_if::Bool = false
     disallow_single_arg_nesting::Bool = false
+    preserve_single_line_if::Bool = true
```

Comment:
should default to `false` to keep current behaviour by default

Reply:
Good point! Done